### DISCUSSION TITLE
tft35 e3 v3.0.1 problem dual hotend

### DISCUSSION THREAD

**Discussion by lidas1234** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2883] [General]
Hi everyone, I have a problem I wanted to add this tft35 e3 v3.0.1 display on a printer that has two separate hotends (chimera) but I can't see the two hotends on the display, instead on the original display I see the two hotends, can anyone help me , Thank you

**Comment by kisslorand** [comment]
Did you check the following section in "config.ini?
```
#### Hotend Count
#   Value range: [min: 0, max: 6]
hotend_count:1
```

**Comment by lidas1234** [comment]
Yes, I checked and changed with
hotend_count:2
